Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -567,6 +570,9 @@ void CGPathAddArcToPoint(
 
 /**
  @Status Interoperable
+ @Notes For the scenario of drawing a full circle, any distance of 2pi will result in a circle being drawn
+        regardless of the direction being drawn. The reference platform however, will not draw a circle
```

Comment:
this is too wordy, and could be trimmed down to a succinct

`All arcs from 0 to 2pi in either direction will result in a circle; the reference platform, by contrast, does not honor this behavior.`